src/aspire/commands/apple.py

```diff
@@ -81,9 +83,12 @@ def apple(
     response_thresh_norm_factor,
     conv_map_nthreads,
     n_processes,
+    loglevel,
 ):
     """Pick and save particles from one or more mrc files."""
 
+    logger.setLevel(getattr(logging, loglevel))
```

Comment:
I think this line and all the others like it does not do what you think.

Reply:
You're totally right. I realized the important thing is setting the level of the `StreamHandler` of the *root* logger. Added utility functions for this. Tested it a few times and works (in particular, all messages still sent to log file)